researchstudio-sat/webofneeds
# Feat svgloadspinner

blocked by #2166 

fixes #782 
fixes #1823 

- removes all existing gif spinners (incl their folder images/spinner/)
- adds a svg animation icon (#ico_loading_anim) that was generated with https://loading.io/#
- the icon colors where then replaced with our variable filling --local-primary and --local-secondary, so we can style both colors
- the styling itself is within the preexisting .hspinner class within the won.scss

How To Test:
-  click on "Load Previous Messages" button in an open conversation should show the spinner
- connection/ws loss should also show the spinner in the slide-in once reconnectt button has been clicked

Disclaimer:
i do not have an opinion about the svg animation, nor do i care about the style or color of it. So if there is a need in changing any of these things then please do so within this PR without re-iterating the ticket process.
